repository/src/main/java/org/apache/atlas/discovery/EntityLineageService.java

```diff
@@ -195,6 +201,67 @@ public AtlasLineageOnDemandInfo getAtlasLineageInfo(String guid, LineageOnDemand
         return ret;
     }
 
+    @Override
+    @GraphTransaction
+    public AtlasLineageListInfo getAtlasLineageListInfo(String guid, LineageListRequest lineageListRequest) throws AtlasBaseException {
+        AtlasPerfMetrics.MetricRecorder metricRecorder = RequestContext.get().startMetricRecord("getAtlasListInfo");
+
+        RequestContext.get().setRelationAttrsForSearch(lineageListRequest.getRelationAttributes());
+
+        AtlasLineageListInfo ret;
```

Comment:
variable assignment can be combined with line # 217

Reply:
Resolved.